test/lit/passes/dce-eh.wast

```diff
@@ -0,0 +1,126 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;; RUN: wasm-opt %s --dce -all -S -o - | filecheck %s
+
+;; Exception handling instruction support
```

Comment:
```suggestion
```

This section header comment probably isn't necessary since the whole file is testing EH support now.